neuraxle/metaopt/random.py

```diff
@@ -40,11 +40,15 @@ def __init__(self, scoring_function=r2_score, joiner=NumpyConcatenateOuterBatch(
         self.scoring_function = scoring_function
         self.joiner = joiner
 
+    def set_step(self, step):
+        MetaStepMixin.__init__(self, step)
+        return self
```

Comment:
set_step was already properly defined in MetaStepMixin, I don't see the point in overriding it here. Plus, it's bad practice to call the constructor again but in a different method that isn't a constructor, this shouldn't be allowed to compile in Python.